assume_cli/cli.py

```diff
@@ -104,6 +104,13 @@ def cli(args=None):
         choices=set(logging._nameToLevel.keys()),
     )
 
+    parser.add_argument(
```

Comment:
is there a description of available parameters for the CLI in the documentation?